pkgs/by-name/ga/game-music-emu/package.nix

```
@@ -1,21 +1,22 @@
 {
   lib,
   stdenv,
-  fetchurl,
+  fetchFromGitHub,
   cmake,
   removeReferencesTo,
   zlib,
 }:
 
 stdenv.mkDerivation rec {
-  version = "0.6.3";
+  version = "0.6.4";
   pname = "game-music-emu";
 
-  src = fetchurl {
-    url = "https://bitbucket.org/mpyne/game-music-emu/downloads/${pname}-${version}.tar.xz";
-    sha256 = "07857vdkak306d9s5g6fhmjyxk7vijzjhkmqb15s7ihfxx9lx8xb";
+  src = fetchFromGitHub {
+    owner = "libgme";
+    repo = "game-music-emu";
+    tag = version;
+    hash = "sha256-XmPcFfKsEe07hH7f0xMs9hRJshOO/p58Zm9fYsmCCoA=";
```

```suggestion
    hash = "sha256-qGNWFFUUjv2R5e/nQrriAyDJCARISqNB8e5/1zEJ3fk=";
```
after using git prefetch for 0.6.4

---

This was picked up after merge and merged in #424058